src/components/Link/ExternalLink.js

```diff
@@ -4,8 +4,12 @@ import classNames from 'classnames'
 
 import { className, styles, LinkInternals } from './shared'
 
-const ExternalLink = ({ button, children, ...linkProps }) => (
-  <a {...linkProps} className={classNames(className, button && 'button')}>
+const ExternalLink = ({ button, children, newTab, ...linkProps }) => (
+  <a
+    {...linkProps}
+    className={classNames(className, button && 'button')}
+    target={newTab ? '_blank' : '_self'}
```

Comment:
External links link we should also have set  `rel="noopener noreferrer"` for privacy and security. See: https://mathiasbynens.github.io/rel-noopener/ https://stackoverflow.com/questions/40593632/use-window-open-but-block-use-of-window-opener

Reply:
Fixed.